CPU Plug-in fails to compile on MSVC2013

Release, x64 without PME or fftw

```
Error	4807	error C2668: 'min' : ambiguous call to overloaded function	platforms\cpu\src\CpuNonbondedForceVec4.cpp	67	1	OpenMMCPU

Error	4813	error C2668: 'min' : ambiguous call to overloaded function	platforms\cpu\src\CpuNonbondedForceVec4.cpp	231	1	OpenMMCPU
```

many times.

Using the 6.3-beta1 tag

--- a/platforms/cpu/src/CpuNonbondedForceVec4.cpp
+++ b/platforms/cpu/src/CpuNonbondedForceVec4.cpp
@@ -24,6 +24,7 @@
 
 #include "SimTKOpenMMUtilities.h"
 #include "CpuNonbondedForceVec4.h"
+#include <algorithm>
 
 using namespace std;
 using namespace OpenMM;

--- a/platforms/cpu/src/CpuNonbondedForceVec8.cpp
+++ b/platforms/cpu/src/CpuNonbondedForceVec8.cpp
@@ -26,6 +26,7 @@
 #include "CpuNonbondedForceVec8.h"
 #include "openmm/OpenMMException.h"
 #include "openmm/internal/hardware.h"
+#include <algorithm>
 
 using namespace std;
 using namespace OpenMM;

